c/driver/postgresql/postgresql_test.cc

```diff
@@ -94,6 +95,33 @@ class PostgresConnectionTest : public ::testing::Test,
   void TestMetadataGetObjectsTablesTypes() { GTEST_SKIP() << "Not yet implemented"; }
   void TestMetadataGetObjectsColumns() { GTEST_SKIP() << "Not yet implemented"; }
 
+  void TestMetadataGetTableSchema() {
+    // We are overriding to test against SQL injection
+    // TODO: should refactor to provide general pattern for all drivers to test
```

Comment:
You can just add

```cpp
TEST_F(PostgresConnectionTest, MetadataGetTableSchemaInjection) {
  // ...
}
```

below instead of overriding here

Reply:
Very nice. Think I've done this now, but wanted to point out that the parent class of this makes a call like 

`ASSERT_NO_FATAL_FAILURE(IngestSampleTable(&connection, &error));` 

while I had to change that to 

`ASSERT_THAT(quirks()->EnsureSampleTable(&connection, "bulk_ingest", &error), IsOkStatus(&error));` 

There's some duplication of features which I'm guessing a refactor of the fixture could solve, but didn't want to try and tackle that now